packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -288,93 +284,81 @@ export class AnalyzerService {
 
         this._backgroundAnalysisProgram.setConfigOptions(configOptions);
 
-        this._executionRootPath = realCasePath(
-            combinePaths(commandLineOptions.executionRoot, configOptions.projectRoot),
-            this.fs
-        );
+        this._executionRootUri = configOptions.projectRoot;
```

Comment:
What's the difference between `executionRoot` and `projectRoot`? Why aren't we combining them anymore?

Reply:
URIs are never relative. So you can't combine them anymore. project root gets computed from the execution root now.